docs/clearlinux-installation-guide.md

```diff
@@ -0,0 +1,55 @@
+# Installing Clear Containers 3.0 on Clear Linux\*
+
+Clear Containers 3.0 is available in Clear Linux since version **17890**.
+
+*Note:
```

Comment:
Is this missing an asterisk at the end?

Reply:
I'd like to have these asterisks removed as we do not use italics for notes in other CC docs.